src/test-run/client-messages.js

```diff
@@ -0,0 +1,8 @@
+// TODO: move this file to the 'client' dir
+// This should be placed in the new client runner directory. Now we have just legacy client runner.
+// Now we leave this file here and will move it when we are starting to implement the new client runner.
```

Comment:
+// TODO: move this file to the 'client' dir.
+// It should be placed in the new client runner directory. Currently, we have only the legacy client runner.
+// We leave this file here and will move it when we start to implement the new client runner.